runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -7326,6 +7326,9 @@ reservationLockEnter(TR::Node *node, int32_t lwOffset, TR::Register *objectClass
       generateRIInstruction  (cg, addImmOp, node, monitorReg, (uintptr_t) LOCK_INC_DEC_VALUE);
       generateRXInstruction(cg, storeOp, node, monitorReg, generateS390MemoryReference(objReg, lwOffset, cg));
       }
+#if defined(TR_TARGET_64BIT) && (JAVA_SPEC_VERSION >= 19)
+   generateSIInstruction(cg, TR::InstOpCode::AGSI, node, generateS390MemoryReference(cg->getMethodMetaDataRealRegister(), fej9->thisThreadGetOwnedMonitorCountOffset(), cg), 1);
+#endif
```

Comment:
Can you update the comment at line 7297 to reflect this AGSI as well?  I think it'll be good to make sure the comments for the subsequent sections reflect the monitor counter additions too.

Reply:
I have updated the comment as you suggested (I think when I rebased, force push lost the change, so can not point to change)